x-pack/plugins/ml/public/explorer/explorer_charts/explore_series.test.js

```diff
@@ -0,0 +1,63 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License;
+ * you may not use this file except in compliance with the Elastic License.
+ */
+
+import seriesConfig from './__mocks__/mock_series_config_filebeat';
+
+jest.mock('ui/chrome',
+  () => ({
```

Comment:
there is a mixture of styles for callbacks, some breaking this function to a new line, some on the same line.
Looks like all of the `mock` calls are on two lines, is this intentional?